addon/layers/geojson.js

```diff
@@ -0,0 +1,60 @@
+/**
+  @module ember-flexberry-gis
+*/
+
+import Ember from 'ember';
+import BaseLayer from './-private/base';
+
+/**
+  Class describing WFS layer metadata.
+
+  @class WfsLayer
+  @extends BaseLayer
+*/
+export default BaseLayer.extend({
+  /**
+    Permitted operations related to layer type.
+
+    @property operations
+    @type String[]
+    @default ['edit', 'remove', 'identify', 'search', 'legend']
+  */
+  operations: ['edit', 'remove', 'identify', 'search', 'legend'],
```

Comment:
do you actually have legend for this layer?

Reply:
edit in branche "feature-add-markerCluster-geojson"